tests/sentry/digests/test_utilities.py

```diff
@@ -69,81 +66,28 @@ def test_get_event_from_groups_in_digest(self):
             e.event_id for e in events
         }
 
-    def test_team_actors_to_user_ids(self):
```

Comment:
Why remove this test? Don't we still want to send digests to users?

Reply:
`team_actors_to_user_ids()` no longer exists.